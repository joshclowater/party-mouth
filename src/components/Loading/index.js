import styled, { keyframes } from 'styled-components';

const pulse = keyframes`
  0% { transform: scale(0); }
  100% {
    transform: scale(1);
    opacity: 0;
  }
`;

export default styled.div`
  width: 10vh;
  height: 10vh;
  margin: 0 auto;
  background-color: #333;
  border-radius: 100%;
  animation: ${pulse} 1s infinite ease-in-out;
`;
