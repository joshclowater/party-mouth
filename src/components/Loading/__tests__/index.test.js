import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Loading from '../';

it('Renders correctly', () => {
  expect(shallow(<Loading />)).toMatchSnapshot();
});
