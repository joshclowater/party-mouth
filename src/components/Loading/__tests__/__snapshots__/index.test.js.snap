// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
.c0 {
  width: 10vh;
  height: 10vh;
  margin: 0 auto;
  background-color: #333;
  border-radius: 100%;
  -webkit-animation: cpyDft 1s infinite ease-in-out;
  animation: cpyDft 1s infinite ease-in-out;
}

<div
  className="c0"
/>
`;
